### DISCUSSION TITLE
vaultwarden compiled on Alpine Linux segfaults without  "RUSTFLAGS=-C target-feature=-crt-static"

### DISCUSSION THREAD

**Discussion by DerVerruckteFuchs** [dani-garcia/vaultwarden#2515] [Ideas]
I've attempted to build vaultwarden on Alpine Linux 3.15 and 3.16, and the resulting binary segfaults when trying to run it. It seems to be a [build environment issue with OpenSSL](https://users.rust-lang.org/t/sigsegv-with-program-linked-against-openssl-in-an-alpine-container/52172). The build environment fix (using `RUSTFLAGS=-C target-feature=-crt-static`) from the link worked for me. I used `env RUSTFLAGS=-C target-feature=-crt-static cargo build...`, and I have a build that runs successfully. It would probably be a good idea to set this somewhere for Alpine Linux builds.

**Comment by BlackDex** [comment]
We do not use alpine as a build environment, so no need to add this. Maybe adding it to the wiki is an option.
